Course-Structure-RDD.md

```diff
@@ -0,0 +1,841 @@
+# Sparkly Course Structure - Read Driven Development Document
+
+> **"Structured Learning, Simplified Delivery"**
+
+## 📋 Executive Summary
+
+This document defines the standardized course structure and manifest system for the Sparkly platform. Our goal is to create a portable, scalable, and developer-friendly format for packaging educational content that can be deployed anywhere from local file systems to cloud platforms.
+
+**Status**: 🏗️ Design Phase  
+**Target Completion**: Q4 2025  
+**Current Focus**: Core manifest schema and folder structure definition
+
+## 🎯 Problem Statement
```

Comment:
Let's regroup problem statement and solution into just one list of features. You could still have a problem statement, but make it more like:

Current courses lack 1:1 individualized custom mentor-like interactions.

Reply:
We should probably tighten up our product manager script to emphasize the need to avoid duplication, filler, and fluff. It seems like no matter how much I mention "Concise, keep it simple, etc.." the AI still needs more guidance to do it well. You may want to use a 2-pass process: Generate, then trim, consolidate, and clean up.